modules/inference-service/main.tf

```diff
@@ -132,8 +132,8 @@ resource "kubernetes_deployment" "inference_deployment" {
           }
 
           volume_mount {
-            mount_path = "/data"
-            name       = "data"
+            mount_path = "/tmp"
```

Comment:
Since this `mounth_path` is tightly coupled with the container image used. Can we add some comment or details around why it needs to be exactly `/tmp`. This would avoid someone changing it later resulting into things not working ?